setup.py

```diff
@@ -445,7 +445,8 @@ def run(self):
 # reasons of simplicity.
 
 def prebuild_libchtslib(ext, force):
-    if HTSLIB_MODE not in ['shared', 'separate']: return
+    if HTSLIB_MODE not in ['shared', 'separate']:
+        return
```

Comment:
I believe judicious use of single-line conditionals improves readability, and I would thank you to leave instances like this as they are.

If E701 warns for lines like this, I suggest you leave E701 deactivated. Note that PEP-8 admits that single-line conditionals are sometimes appropriate.